edge/client/interface.go

```diff
@@ -25,7 +25,13 @@ type PollStatusResponse struct {
 	CheckinInterval float64          `json:"checkin"`
 	Credentials     string           `json:"credentials"`
 	Stacks          []StackStatus    `json:"stacks"`
-	AsyncCommands   []AsyncCommand   `json:"commands"` // async mode only
+
+	// Async mode only
```

Comment:
I think those should go under a nested struct `AsyncSettings` so that it's obvious. this comment is only visible here, it won't be visible for devs in other files

Reply:
I agree but I'll probably refactor this and other things separately from this PR.